inference-engine/src/gna_plugin/gna_executable_network.hpp

```diff
@@ -59,12 +59,13 @@ class GNAExecutableNetwork : public InferenceEngine::ExecutableNetworkThreadSafe
         return std::make_shared<GNAInferRequest>(plg, networkInputs, networkOutputs);
     }
 
-
-
+    INFERENCE_ENGINE_DEPRECATED("Use InferRequest::QueryState instead")
     std::vector<InferenceEngine::IMemoryStateInternal::Ptr>  QueryState() override {
```

Comment:
can we replace `IMemoryState*` with `IVariableState*` everywhere? except `using IMemoryState = IVariableState;`

Reply:
replaced *MemoryState* to *VariableState* everywhere (except constructions like IMemoryState=IVariableState of course)